europe/italy/rome.md

```diff
@@ -0,0 +1,6 @@
+# Rome
+
+I wish to go to [Rome](https://en.wikivoyage.org/wiki/Rome) because I want to have [pasta](https://www.instagram.com/pastationlondon/?hl=en) for my lunch
```

Comment:
Hyperlinks should be more descriptive as where you are going.
```suggestion
I wish to go to Rome ([wikivoyage](https://en.wikivoyage.org/wiki/Rome)) because I want to have [pasta](https://www.instagram.com/pastationlondon/?hl=en) for my lunch
```